### DISCUSSION TITLE
Expose `SelectComponents` type.

### DISCUSSION THREAD

**Discussion by jimmy-guzman** [JedWatson/react-select#5988] [Ideas]
Is there any appetite in exposing the [SelectComponents](https://github.com/JedWatson/react-select/blob/master/packages/react-select/src/components/index.ts#L51) signature? Right now we have to import from `node_modules/react-select/dist/declarations/src/components` when implementing custom components.

**Comment by Methuselah96** [comment]
We're exporting `SelectComponentsConfig`, is that not sufficient?
